websites/N/Notespace/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "630812692659044352",
+		"name": "itsmateo20"
+	},
+	"service": "Notespace",
+	"description": {
+		"en": "Notespace is your learning platform. Here you will find tons of notes, videos and much more, all for free!",
+		"pl": "Notespace to twoja plaforma do nauki. Znajdziesz tutaj mnóstwo notatek, filmów i wiele więcej, a to wszystko za darmo!"
+	},
+	"url": "notespace.edu.pl",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/W7Ybot8.png",
+	"thumbnail": "https://i.imgur.com/Xw72bq6.png",
+	"color": "#111111",
+	"category": "other",
+	"tags": [
+		"learning",
+		"e-learning",
+		"notes",
+		"exams"
+	],
+	"iframe": false,
+	"readLogs": false
```

These fields default to `false`, so they can be removed safely.
```suggestion
	]
```